dali/pipeline/operators/reader/coco_reader_op.h

```diff
@@ -73,19 +61,11 @@ class COCOReader : public DataReader<CPUBackend, ImageLabelWrapper> {
   }
 
  protected:
-  void ParseAnnotationFiles() {
-    ParseAnnotationFilesHelper(annotations_filename_, annotations_multimap_,
-                                image_id_pairs_, ltrb_, ratio_, size_threshold_, skip_empty_);
-  }
-
-  std::vector<std::string> annotations_filename_;
   AnnotationMap annotations_multimap_;
-  std::vector<std::pair<std::string, int>> image_id_pairs_;
-  bool ltrb_;
-  bool ratio_;
-  float size_threshold_;
-  bool skip_empty_;
   bool save_img_ids_;
+  bool shuffle_after_epoch_;
```

Comment:
this could be a local variable in the constructor if you don't need to use it outside of it

Reply:
Done. Also moved other fields.